momento-sdk/src/main/java/momento/sdk/responses/storage/StorageValue.java

```diff
@@ -4,56 +4,56 @@
 
 public class StorageValue {
```

Comment:
Should this be `StorageItem`?

Reply:
This only has the value part of the item (item = (key, value)).